### PR TITLE
Update to new year

### PR DESCRIPTION
**Pull request by danobi**
Wake up and smell the bacon, it's the next year already!

### CODE REVIEW

**Comment by atsci** [comment]
clang format *successful*! https://ci.trafficserver.apache.org/job/clang-format-github/191/

**Comment by atsci** [comment]
RAT check *successful*! https://ci.trafficserver.apache.org/job/RAT-github/206/

**Comment by atsci** [comment]
AU check *successful*! https://ci.trafficserver.apache.org/job/autest-github/188/

**Comment by atsci** [comment]
FreeBSD11 build *successful*! https://ci.trafficserver.apache.org/job/freebsd-github/1887/

**Comment by atsci** [comment]
Intel CC build *successful*! https://ci.trafficserver.apache.org/job/icc-github/317/

**Comment by atsci** [comment]
Linux build *successful*! https://ci.trafficserver.apache.org/job/linux-github/1778/

**Comment by atsci** [comment]
clang-analyzer build *successful*! https://ci.trafficserver.apache.org/job/clang-analyzer-github/450/

File: doc/conf.py [hunk 1]
```diff
@@ -81,7 +81,7 @@
 
 # General information about the project.
 project = u'Apache Traffic Server'
-copyright = u'2016, dev@trafficserver.apache.org'
+copyright = u'2017, dev@trafficserver.apache.org'
```
Comment (mlibbey) [inline doc/conf.py]: Should we just import one of the date/time modules and grab the year from there?

Comment (danobi) [inline doc/conf.py]: Sounds like a good idea to me. I did a quick look into the legal aspects of this and it seems fine.

Comment (SolidWallOfCode) [inline doc/conf.py]: I like it. Make it so, Danobi.

**Comment by danobi** [comment]
Yes master

**Comment by atsci** [comment]
clang format *successful*! https://ci.trafficserver.apache.org/job/clang-format-github/198/

**Comment by atsci** [comment]
RAT check *successful*! https://ci.trafficserver.apache.org/job/RAT-github/212/

**Comment by atsci** [comment]
FreeBSD11 build *successful*! https://ci.trafficserver.apache.org/job/freebsd-github/1893/

**Comment by atsci** [comment]
AU check *successful*! https://ci.trafficserver.apache.org/job/autest-github/194/

**Comment by atsci** [comment]
Intel CC build *successful*! https://ci.trafficserver.apache.org/job/icc-github/323/

**Comment by atsci** [comment]
Linux build *successful*! https://ci.trafficserver.apache.org/job/linux-github/1784/

**Comment by atsci** [comment]
clang-analyzer build *successful*! https://ci.trafficserver.apache.org/job/clang-analyzer-github/456/

**Pull request merged**